c2corg_ui/templates/article/edit.html

```diff
@@ -0,0 +1,156 @@
+                    </div>
+                  </div>
+                </div>
+              </div>
+
+            </section>
+
+            <div class="form-group half half-group">
+
+              <div ng-class="{ 'has-success': article.article_type}">
+                {{article.article_type}}
+                <label><span translate>article_type</span> *</label>
+                % if updating_doc:
+                  <select ng-if="userCtrl.auth.isModerator()" class="form-control" ng-model="article.article_type"
+                          ng-options="type as mainCtrl.translate(type) for type in ${article_types}" ><option value=""></option></select>
+                  <select ng-if="userCtrl.auth.isModerator() == false && article.article_type =='collab'"
+                          class="form-control" ng-model="article.article_type"
+                          ng-options="type as mainCtrl.translate(type) for type in ${article_types} | filter: 'collab'" ><option value=""></option>
+                  </select>
+                  <select ng-if="userCtrl.auth.isModerator() == false && article.article_type =='personal'"
+                          class="form-control" ng-model="article.article_type"
+                          ng-options="type as mainCtrl.translate(type) for type in ${article_types}" ><option value=""></option>
+                  </select>
+                % else:
+                  <select class="form-control" ng-model="article.article_type" ng-options="type as mainCtrl.translate(type) for type in ${article_types} " ><option value=""></option></select>
+                % endif
+                  <span class="glyphicon glyphicon-ok form-control-feedback" ng-show="article.article_type"></span>
+              </div>
+
+              ## ARTICLE CATEGORIES
+              <div ng-class="{'has-success': article.article_categories}" ng-init="articleCategories = ${article_categories}">
+                <label><span translate>article_categories</span> *</label>
+                <select ng-options="type as mainCtrl.translate(type) for type in articleCategories" ng-model="article.article_categories" class="form-control"><option value=""></option></select>
+                <span class="glyphicon glyphicon-ok form-control-feedback" ng-show="article.article_categories"></span>
+              </div>
+
+            </div>
+          </div>
+        </section>
+
+        <section class="section">
+          <h2 class="heading show-phone" translate>description</h2>
+          <div class="content">
+
+            ## SUMMARY
+            <div id="summary-group" class="form-group full">
+              <label translate>summary</label>
+              <textarea name="summary" ng-model="article.locales[0].summary" class="form-control"></textarea>
+            </div>
+
+            ## DESCRIPTION
+            <div id="description-group" class="form-group full">
+              <label translate>article core</label>
```

Comment:
"Article body" sounds good, thanks.
The fieldname is useful in the editing form, not in the detail view, I think.